### ISSUE TITLE
Crash the app and dont know how to debbug

### ISSUE THREAD

**Issue by xor908** [Akito13/plumbly#2]
Hello,

I use plumble, and some times the app crash and when i open again need to reconnect.

I dont know how to debbug or how view the log of app. I will try with Android Studio.

If someone have a tip for that i will be very happy.

Maybe the problem is on the phone, I don't really know.

Sorry for my bad english :(

Thanks a lot!!

**Comment by theAkito** [comment]
@xor908 
Greetings! Thank you for your contribution.

Currently, we are still working on kickstarting this fork properly. Once we have fixed the fundamental issues with this project, we will test it thoroughly and take your issue into account, specifically, as well. 

If we get it done quickly, maybe after this weekend we will have a working version of this project again, so we can properly debug it. By then we will address your issue.

For now, please be a little patient, thank you.

**Comment by quite** [comment]
@akito13 I made an attempt at getting plumbly to build (and start) in modern Android Studio. In branch `buildit` here: https://github.com/quite/plumbly (which has the submodule jumbly from the same-name branch of https://github.com/quite/jumbly).

It builds, but it doesn't really work. Can't seem to hear, nor transmit audio (in any transmission mode; in push-to-talk the button doesn't budge...). Haven't debugged further.

**Comment by theAkito** [comment]
@quite Thank you for your contribution! If you like you can create a pull request on a non-master branch. However, I was told @iammeat had already a more or less working version running, but this upgrade wasn't yet applied to this main repository.

**Comment by quite** [comment]
If @iammeat has something which actually is usable, I think that whould be a better starting point :) But let's see first if they respond at all...

**Comment by theAkito** [comment]
@quite Agreed. At the time when @iammeat was working on it, he was pretty busy, but maybe something has improved by now, at least I hope so. 😃

**Comment by quite** [comment]
ping @iammeat ... would be swell to have a fresher and more usable android client to mumble!

**Comment by quite** [comment]
I forked Plumble myself. It is called Mumla (that's Mumble in Swedish). It lives here: https://gitlab.com/quite/mumla  (Jumbly became *humla*, that's bumblebee :) @rock2rap
You're welcome to help out!

There is a build available here: https://lublin.se/mumla/

**Comment by deluxghost** [comment]
@quite looking good, might be the first public release atm.

something I wish: full mumble 1.3.0 features (even admin features), localization, modern android ui

**Comment by quite** [comment]
I'm all for those things. And one way is to step by step built this into Mumla. Mumla/Plumble feels like a solid ground to build upon, even if there is a lot of weeding and refactoring to be done to for example modernize the UI. Hopefully see you over at https://gitlab.com/quite/mumla :) Contributions of any kinds welcome!
